Makefile

```diff
@@ -35,10 +35,13 @@ black_fix: test_build
 code_format: black_fix
 	echo "Code Formatted"
 
+check_checks: test_build
+	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- python manage.py check"
+
 check_migrations: test_build
 	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- python manage.py makemigrations --check"
 
-check: test_build check_migrations black_check lint test
+check: test_build check_migrations check_checks black_check lint test
```

Comment:
Actually now that I think about it I don't think this belongs here at all.  This is statically checking configuration of your local development environment, it should probably go under `make up`, not here under the CI checks?

Reply:
Won't it run these static checks when the dev server starts up?

Reply:
Checks are automatically run when you run `./manage.py migrate` and `./manage.py runserver`. But if you're not doing any migrations, it won't run.

Reply:
That the checks are done during `migrate` is good because it ultimately checks that the env vars set up by CloudOps are correct/sane. I don't know exactly when the `migrate` command is executed but the fact that it does means that we probably raise awareness before rolling it out. 

However, we don't use `./manage.py runserver` in CI and but I believe we do execute all the migrations during `pytest` so that'll check stuff in CI. 

In other words, I agree with you. We shouldn't be mentioned ANYWHERE in the Makefile and or `.circleci/config.yml` (or any of its descendents). It's run anyway.